modules/custom/az_paragraphs/az_paragraphs.info.yml

```diff
@@ -4,4 +4,7 @@ description: 'Provides custom paragraph types.'
 core_version_requirement: ^8.8 || ^9
 package: 'The University of Arizona'
 dependencies:
+  - az_flexible_page
```

Comment:
Just curious: is there a specific reason `az_flexible_page` needs to be a dependency here?  Looks like `az_paragraphs` already a dependency for `az_flexible_page` so I'm not sure we want/need the circular dependency (?)

Reply:
I believe I put this due attempting to resolve a configuration dependency issue in the provided configuration of the entity embed to pages (this module provides the entity_embed configuration to pages and so pages have to exist).

However now that I think about it, this dependency here failed to resolve the entity embed configuration dependency issue (it was still complaining at install time that the configuration didn't exist) and it was only resolved by making the embed configuration optional rather than install.

Good catch on this being a circular dependency...since this dependency didn't seem to actually resolve the config dependency issue for the page embed maybe this can just be removed, since the actual fix seemed to be flagging the entity_embed configuration as optional so it can be installed after the fact.

Reply:
It looks like it works fine without this. I think it'd be good to remove it.